credentials/test/operational-certificates-error-cases/Chip-Test-ICAC-Ext-AKID-Length-Invalid-Key.chip

```diff
@@ -0,0 +1 @@
+N�AWS�ψj*r��S�IA�s�� ���F�k�j�{@�E�"pN� �O�^+��2(DU�V��dA�����6��:��H#aM��SvcJ�a޽�QIj
```

Comment:
Github shows a diff here, so I am guessing it thinks this is text.

Should we add `.gitattributes` to treat `*.chip binary` ?

Reply:
added